specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -0,0 +1,208 @@
+      "version": "7.4-preview.1"
+    },
+    "x-ms-parameterized-host": {
+      "hostTemplate": "{vaultBaseUrl}",
+      "useSchemePrefix": false,
+      "positionInOperation": "first",
+      "parameters": [
+        {
+          "name": "vaultBaseUrl",
+          "description": "The vault name, for example https://myvault.vault.azure.net.",
+          "required": true,
+          "type": "string",
+          "in": "path",
+          "x-ms-skip-url-encoding": true
+          
+        }
+      ]
+    },
+    "consumes": [
+      "application/json"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "paths": {
+      "/settings/{setting-name}": {
+        "patch": {
+          "tags": [
+            "Settings"
+          ],
+          "operationId": "UpdateSettings",
+          "summary": "Updates key vault account setting, stores it, then returns the setting name and value to the client.",
+          "description": "If false, all key management operations must happen through the Managed HSM data plane. If true, allows users to manage keys via the Azure Resource Manager.",
+          "parameters": [
+            {
+              "name": "setting-name",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the account setting. Must be a valid settings option."
+            },
+            {
+              "name": "parameters",
+              "in": "body",
+              "required": true,
+              "x-ms-client-flatten": true,
+              "schema": {
+                "$ref": "#/definitions/UpdateSettingsRequest"
+              },
+              "description": "The parameters to update an account setting."
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "An account setting containing the name of the setting and the value it is set to. ",
+              "schema": {
+                "$ref": "#/definitions/Setting"
+              }
+            },
+            "default": {
+              "description": "Key Vault error response describing why the operation failed.",
+              "schema": {
+                "$ref": "common.json#/definitions/KeyVaultError"
+              }
+            }
+          },
+          "x-ms-examples": {
+            "Create key": {
+              "$ref": "./examples/UpdateSetting-example.json"
+            }
+          }
+        },
+        "get": {
+          "tags": [
+            "Settings"
+          ],
+          "operationId": "GetSettingValue",
+          "summary": "Get specified account setting value.",
+          "description": "Retrieves the value of a specified, value account setting.",
+          "parameters": [
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
```

Comment:
As named and described, this gets a single setting value but takes no setting name. You should define the "setting-name" parameter in the "parameters" section of this file (add it) and `$ref` it in both places.